js/output/webpage/webpage-output.js

```diff
@@ -252,6 +252,7 @@ window.WebpageOutput = Backbone.View.extend({
         _.extend(this.tester.testContext, {
             $doc: $(this.frameDoc),
             docSP: this.slowparseResults.document,
+            $docSP: $(this.slowparseResults.document),
```

Comment:
I'm not sure if this just changed recently or what - but all of my webpage challenges were failing locally because this was missing, so I added it in?

Reply:
This did change recently (as I removed jQuery from webpage-output.js), and there were a few webpage challenges that I updated (like Write a Poem). Does your localhost have the latest challenges?

Reply:
Ahh - I must not have the latest challenges locally then. I could remove this or keep it in - what would you prefer? (Keeping it in feels safer in the near term, at least until we fully transition away from jQuery.)

Reply:
It's been out of the live codebase for a few months, so I'd vote for keeping it removed. The challenges should work on ZND.